addons/pos_sale/views/sales_team_views.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="crm_team_view_form_inherit_pos_sale" model="ir.ui.view">
+        <field name="name">crm.team.form</field>
+        <field name="model">crm.team</field>
+        <field name="inherit_id" ref="sales_team.crm_team_view_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='kanban_dashboard_graph_model']" position="attributes">
+                <attribute name="attrs">{'invisible': [('team_type', '=', 'pos')]}</attribute>
+            </xpath>
+        </field>
+    </record>
+
+    <act_window
+        id="action_sessions_list"
+        name="Orders"
+        src_model="crm.team"
+        res_model="pos.session"
+        view_type="form"
+        view_mode="tree,form"
+        context="{'search_default_open_sessions': True}"
+        domain="[('crm_team_id', '=', active_id)]"/> <!-- How to put this as filter  -->
```

Comment:
define it in the search view, and set a key context `search_default_<filter_name>` ;)